### PR TITLE
[Messenger] Add worker metadata inside logs

### PR DESCRIPTION
**Pull request by 94noni**
| Q             | A
| ------------- | ---
| Branch?       | 5.4 
| Bug fix?      | no
| New feature?  | .
| Deprecations? | no
| Tickets       | Improve https://github.com/symfony/symfony/pull/42723
| License       | MIT
| Doc PR        | .

Small PR related to one of my comment https://github.com/symfony/symfony/pull/42723#discussion_r695985273 on the ticket PR: having this contextual info is valuable imho
cc @ruudk @okwinza @fabpot

### CODE REVIEW

**Review by ruudk — APPROVED** [review]
Nice!

**Review by okwinza — APPROVED** [review]
:+1:

**Review by lyrixx — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

File: src/Symfony/Component/Messenger/EventListener/StopWorkerOnSigtermSignalListener.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ public function onWorkerStarted(WorkerStartedEvent $event): void
     {
         pcntl_signal(\SIGTERM, function () use ($event) {
             if (null !== $this->logger) {
-                $this->logger->info('Received SIGTERM signal.');
+                $this->logger->info('Received SIGTERM signal.', ['transportNames' => $event->getWorker()->getMetadata()->getTransportNames()]);
```
Comment (fabpot) [inline src/Symfony/Component/Messenger/EventListener/StopWorkerOnSigtermSignalListener.php]: I think we are using underscore names in such a situation, so `transport_names`.

Comment (94noni) [reply src/Symfony/Component/Messenger/EventListener/StopWorkerOnSigtermSignalListener.php]: @fabpot in fact, I've found both cases `camelCase`|`snake_case` in the codebase
I'll rework this but maybe it is needed a consensus here :)

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @94noni.

**Pull request merged**